experimental/GModule/GModule.jl

```diff
@@ -3,6 +3,142 @@ using Oscar
 using Hecke
 import Hecke: data
 
+
+"""
+    restriction_of_scalars(M::GModule, phi::Map)
+
+Return the `S`-module obtained by restricting the scalars of `M`
+from `R` to `S`, where `phi` is an embedding of `S` into `R`.
+
+If `R` has `S`-rank `d` and `M` has rank `n` then the returned module
+has rank `d*n`.
+
+# Examples
+
```

Comment:
Do you plan to add these examples soon? Or should we just add "TODO" and merge it?